### DISCUSSION TITLE
[UX] Do not allow to close global filter if there is a value

### DISCUSSION THREAD

**Discussion by zaverden** [KevinVandy/material-react-table#63] [Ideas]
# Problem
At the moment it is possible to have global filter closed even if it's not empty. this is not a good user experience. for example, if I have global filter from query string and send tht link to mates - they'll see filtered data but will be unaware there is a filter. only if developer did not forget to set `initialState.showGlobalFilter` based on query

# Proposal
Do not allow to close global filter if there is a value (have it always open)

I implement it if this proposal feels reasonable

**Comment by KevinVandy** [comment]
Good idea. So disable the toggle button when a filter is present, or clear the global filter when the search box is hidden?

**Reply by KevinVandy** [reply]
The toggle search button is now disabled when a global filter is present as of beta.9
